images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -236,32 +242,32 @@ function Get-PowerShellAzureModules {
 
     [Array] $azInstalledModules = Get-ChildItem -Path "C:\Modules\az_*" -Directory | ForEach-Object { $_.Name.Split("_")[1] }
     if ($azInstalledModules.Count -gt 0) {
-        $result += "Az: $($azInstalledModules -join ', ')"
+        $result += [ToolVersionsListNode]::new("Az", $($azInstalledModules), '^\d+', "Inline")
     }
 
     [Array] $azureInstalledModules = Get-ChildItem -Path "C:\Modules\azure_*" -Directory | ForEach-Object { $_.Name.Split("_")[1] } | ForEach-Object { if ($_ -eq $defaultAzureModuleVersion) { "$($_) (Default)" } else { $_ } }
     if ($azureInstalledModules.Count -gt 0) {
-        $result += "Azure: $($azureInstalledModules -join ', ')"
+        $result += [ToolVersionsListNode]::new("Azure", $($azureInstalledModules), '^\d+', "Inline")
     }
 
     [Array] $azurermInstalledModules = Get-ChildItem -Path "C:\Modules\azurerm_*" -Directory | ForEach-Object { $_.Name.Split("_")[1] } | ForEach-Object { if ($_ -eq $defaultAzureModuleVersion) { "$($_) (Default)" } else { $_ } }
     if ($azurermInstalledModules.Count -gt 0) {
-        $result += "AzureRM: $($azurermInstalledModules -join ', ')"
+        $result += [ToolVersionsListNode]::new("AzureRM", $($azurermInstalledModules), '^\d+', "Inline")
     }
 
     [Array] $azCachedModules = Get-ChildItem -Path "C:\Modules\az_*.zip" -File | ForEach-Object { $_.Name.Split("_")[1] }
     if ($azCachedModules.Count -gt 0) {
-        $result += "Az (Cached): $($azCachedModules -join ', ')"
+        $result += [ToolVersionsListNode]::new("Az (Cached)", $($azCachedModules), '^\d+', "Inline")
     }
 
     [Array] $azureCachedModules = Get-ChildItem -Path "C:\Modules\azure_*.zip" -File | ForEach-Object { $_.Name.Split("_")[1] }
     if ($azureCachedModules.Count -gt 0) {
-        $result += "Azure (Cached): $($azureCachedModules -join ', ')"
+        $result += [ToolVersionsListNode]::new("Azure (Cached)", $($azureCachedModules), '^\d+', "Inline")
```

Comment:
I think we should use `^\d+\.\d+` regex for Azure modules

Reply:
agree